core/blockchain/impl/block_tree_impl.hpp

```diff
@@ -59,6 +59,8 @@ namespace kagome::blockchain {
      * the operations faster
      */
     struct TreeMeta {
+      TreeMeta(TreeNode &subtree_root_node);
```

Comment:
Hm, doesn't this line get a warning about making the constructor explicit?

Reply:
Agree. Add **explicit**